doc/source/examples/02-subset_assignment.py

```diff
@@ -0,0 +1,46 @@
+"""
+.. _ref_subset_assignment:
+
+Subset assignment
+=================
+
+This example demonstrates how to assign a subset 
+to a training data.
+
+"""
+
+###############################################################################
+# Import necessary libraries
+# --------------------------
+
+import ansys.simai.core as asc
+
+simai = asc.from_config()
+
+###############################################################################
+# Select a training data
+# ----------------------
+# Example of a training_data_id associated with a project_id
+
+training_data_id = "k4z77qzq"
+project_id = "k9756vw0"
+
+###############################################################################
+# Get subset assignment
+# ---------------------
+# Get the current subset assigned for this training_data_id.
+
+simai.training_data.get(id=training_data_id).get_subset(project=project_id)
+
+###############################################################################
+# Assign a new subset (two options)
+# ---------------------------------
+# Manually assign a new subset to the training data.
+
+simai.training_data.get(id=training_data_id).assign_subset(project=project_id, subset="Validation")
```

Comment:
Actually we plan to get rid of validation subset for users so probably use one of the other ones instead.
```suggestion
simai.training_data.get(id=training_data_id).assign_subset(project=project_id, subset="Test")
```

Reply:
@tmpbeing I actually discussed this with Pierre Y. But he thought that it could still be useful to some users to provide the example including the validation subset as we do not know when it will be merged into the training subset. 

Do you have an idea as to when this "merge" will occur? Depending on how long it takes, it might be worth keeping the example as is and altering it later on(?)